tripleblindmarket/covid-safe-paths
# Build time feature flags (Phase 1)

Introduces build time feature flagging. Flags are build time, and cannot change at runtime yet (this will be added later)

- Uses https://github.com/garbles/flag for React feature flagging.
- Uses `react-native-config` to read the `.env` and parse `flag_*=true` vars into flag booleans
- Provides a `.env.release` which should allow configuration of flags for release only

#### Linked issues:

None

#### Screenshots:

None

#### How to test

* `yarn run-android` to see that Google import is available in settings
* Build an android release version to check that it is now disabled
